package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "courageous-trapeze",
+  "version": "1.0.0",
+  "description": "Courageous Trapeze allows you to import your Google contacts to schedule text and social communications with your closest contacts.",
+  "main": "index.js",
+  "scripts": {
+    "test": ""
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/onelovelyname/courageous-trapeze.git"
+  },
+  "author": "John Yeglinski, Kevin Huang, Mike Harris, Melanie Gin",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/onelovelyname/courageous-trapeze/issues"
```

Comment:
Let's use the upstream's URL path instead of your fork's path...same for lines 11 and 18